binary/README.md

```diff
@@ -0,0 +1,13 @@
+# cardano-sl-binary
+
+Binary serialisation and deserialisation for Cardano SL.
+
+This library is built on top of the [binary] and [cborg] packages and implement
+space efficient serialising and deserialising of Haskell data types to and from
+binary representations via a `Bi` type class and associated machinery.
+
+The library includes ability the derive `Bi` instances (requires a `Typeable`
+constraint).
+
+[binary]: https://hackage.haskell.org/package/binary
```

Comment:
I can see it's there. Are those instances still used?

Reply:
Well the `deriveSimpleBi`*  functions are used in at least 6 other packages. Maybe the derived instances are not used, but that's another matter.

Reply:
Are you saying that the `Bi` typeclass shouldn't actually be used?

I started at the top of the tree in `explorer` and tried to remove usage of `deriveSimpleBi` but it seems pretty baked in and still very much in use.

I also noticed that the binary package seems to define two ways of generate `Bi` instances, the TH  method and one using generics. I had a bit of a stab at moving from TH to generics in `explorer` but its non-trivial so I'm dropping that for now.